ecs/spotfleet-cluster/example/main.tf

```diff
@@ -0,0 +1,21 @@
+data "aws_region" "current" {}
+
+module "vpc" {
+  source          = "../../../ec2/vpc"
+  prefix          = "${var.prefix}"
+  cidr_block      = "10.1.0.0/16"
+  tags            = "${var.tags}"
+  private_subnets = "0"
+}
+
+module "example_spotfleet-cluster" {
+  source              = "../"
+  prefix              = "${var.prefix}"
+  target_capacity     = 6
```

Comment:
Please order the variables a bit here. `spot_price` and `target_capacity` and `allocation_strategy` should probably be grouped together.